client/src/components/History/Layout/MultipleView.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <CurrentUser v-slot="{ user }">
+        <UserHistories v-if="user" v-slot="{ currentHistory, histories, handlers, historiesLoading }" :user="user">
+            <div v-if="!historiesLoading" id="histories" class="columns-container d-flex flex-row h-100">
+                <HistoryPanel
```

Comment:
Note, we discussed this further and intend to remove the current panel from the multi-view for now and initially use the existing current history of the right panel. This will make this transition easier and we can always add another history to the left or hide the side panels in a follow-up if necessary.